lib/audit/catalog_to_moab.rb

```diff
@@ -134,7 +134,7 @@ def update_status(new_status)
     end
   end
 
-  def online_moab_found?(druid, storage_dir)
+  def online_moab_found(druid, storage_dir)
```

Comment:
similar to my prior comment, i'd propose either naming this `online_moab`, or keeping it `online_moab_found?` and returning a bool.

Reply:
ill change it to `online_moab` the method is not returning a boolean.